wavesrv/pkg/remote/remote.go

```diff
@@ -164,7 +168,10 @@ type MShellProc struct {
 	RunningCmds      map[base.CommandKey]*RunCmdType
 	PendingStateCmds map[pendingStateKey]base.CommandKey // key=[remoteinstance name] (in progress commands that might update the state)
 
-	Client *ssh.Client
+	Client          *ssh.Client
+	sudoPw          *[]byte
```

Comment:
sudoPw should be a `[]byte` not a `*[]byte`.  it is a slice, so it is still possible to set it to nil.